Scripts/Services/Assistant/V1/Model/DialogNode.cs

```diff
@@ -199,7 +199,7 @@ public class DigressOutSlotsValue
         /// - It must be no longer than 1024 characters.
         /// </summary>
         [JsonProperty("dialog_node", NullValueHandling = NullValueHandling.Ignore)]
-        public string DialogNodeId { get; set; }
+        public string _DialogNode { get; set; }
```

Comment:
why is it `_DialogNode` now?

Reply:
a model cannot have a property with the same name (in this case DialogNode.DialogNode). That is reserved for the constructor. In the generator we append `_` to this. When it was handwritten I just added `Id`